assets/l10n/app_en.arb

```diff
@@ -529,6 +529,10 @@
   "@userRoleUnknown": {
     "description": "Label for UserRole.unknown"
   },
+  "inboxButton": "Inbox",
+  "@inboxButton": {
+    "description": "Label for the menu button switching to inbox."
```

Comment:
Updating this with the description altered to match the other existing page titles.

I guess this would also be a reason to also name the button as "Choose account" for the "Choose account" page.

Reply:
> The documentation for `SemanticsProperties.namesRoute` doesn't seem quite clear on how it can be used. I assume that we need to use `Semantics` with it? Because this might be a non-trivial project on its own, I feel that it might fit better as a separate issue.

Yes, definitely fine to leave this for later 🙂:

> How about adding a `TODO(a11y)` to find out if we're meeting users' expectations when they use Zulip with a screen reader, mentioning `SemanticsProperties.namesRoute`?

You could file an issue if you want, but I think the TODO is fine and better than nothing.

Reply:
I found #535 relevant to this:

> We should try out the app with iOS's VoiceOver and/or Android's TalkBack, and make sure it's possible to do the essentials to be able to use the app at all:

> - get between the different top-level pages (Inbox, All messages, Subscribed streams, Direct messages)

Except that the top-level pages become the tabs on the new home screen. With an app bar it might become less of a concern, but still it would be helpful to experiment with this before launch.